themes/bootstrap5/js/searchbox_controls.js

```diff
@@ -238,6 +239,19 @@ VuFind.register('searchbox_controls', function SearchboxControls() {
           hiddenFilters.push($(this).val());
         });
 
+        if (applyActiveFilters) {
+          // There may be multiple copies of the active-filters area, so be sure to only pull from one:
+          const activeFilters = document.querySelector(".active-filters");
+          if (activeFilters) {
+            activeFilters.querySelectorAll(".filter-value").forEach(
+              (element) => {
+                if (element.dataset.filter) {
+                  hiddenFilters.push(element.dataset.filter);
```

Comment:
One could argue that the hiddenFilters parameter used by the AJAX handler would be more appropriately renamed to filters since from the AC perspective, it doesn't really matter if they're hidden or not... but I opted not to change it here in the interest of minimizing changes.